cmd/nerdctl/compose_run_linux_test.go

```diff
@@ -125,7 +125,7 @@ services:
 		// unbuffer(1) can be installed with `apt-get install expect`.
 		unbuffer := []string{"unbuffer"}
 		base.ComposeCmdWithHelper(unbuffer, "-f", comp.YAMLFullPath(),
-			"run", "--service-ports", "--name", containerName, "web").Run()
+			"run", "--service-ports", "--name", containerName, "web").AssertOK()
```

Comment:
I think it makes sense to `AssertOK()` here. If it fails, the for loop check below will also not work.